examples/specs/normalized/layer_point_errorbar_ci_normalized.vl.json

```diff
@@ -41,12 +41,12 @@
               {
                 "field": "upper_yield",
                 "type": "quantitative",
-                "title": "Upper error of yield"
+                "title": "ci1 of yield"
```

Comment:
ci0 / ci1 are not naturally understood operations (very specific to Vega [for "The lower boundary of the bootstrapped 95% confidence interval of the mean field value."](https://vega.github.io/vega/docs/transforms/aggregate/#ops))

Perhaps, "Mean of <fieldname> (Upper/Lower bound of 95% CI)". 
(Though I think it's still a bit too long.)  

@domoritz  Any suggestion?

Reply:
Actually, if users directly uses ci, they gets this title as well. So perhaps, we can keep it like this in this PR.  

(But make sure that this title thing is generated from a common code (by calling [`title()` in `fielddef.ts`](https://github.com/vega/vega-lite/blob/master/src/fielddef.ts#L524)).  

We can then consider if we should revise default title for `ci0`/`ci1`.